app/controllers/grade_entry_forms_controller.rb

```diff
@@ -382,7 +383,7 @@ def csv_upload
         if totals.empty?
           totals = row
           # Create/update the grade entry items
-          GradeEntryItem.create_or_update_from_csv_rows(
+          grades << GradeEntryItem.create_or_update_from_csv_rows(
```

Comment:
You can just do `grades = ...` here, since `create_or_update_from_csv_rows` returns a list.